### PR TITLE
Cookie initialy baked by NetworkToCode Cookie Drift Manager Tool

### PR DESCRIPTION
**Pull request by snaselj**
Template:

```
{
    "template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
    "dir": "nautobot-app",
    "ref": "develop",
    "path": null
}
```

Cookie:

```
{
    "remote": "https://github.com/nautobot/nautobot-plugin-nornir.git",
    "path": "/opt/ntc/drift-manager/outputs/nautobot-plugin-nornir",
    "repository_path": "/opt/ntc/drift-manager/outputs/nautobot-plugin-nornir",
    "dir": "",
    "branch_prefix": "drift-manager",
    "context": {
        "codeowner_github_usernames": "@smith-ntc",
        "full_name": "Network to Code, LLC",
        "email": "info@networktocode.com",
        "github_org": "nautobot",
        "plugin_name": "nautobot_plugin_nornir",
        "verbose_name": "Nautobot Nornir Plugin",
        "plugin_slug": "nautobot-plugin-nornir",
        "project_slug": "nautobot-plugin-nornir",
        "repo_url": "https://github.com/nautobot/nautobot-plugin-nornir",
        "base_url": "plugin-nornir",
        "min_nautobot_version": "1.4.0",
        "max_nautobot_version": "1.9999",
        "nautobot_version": "latest",
        "camel_name": "NautobotPluginNornir",
        "project_short_description": "Nautobot Nornir Plugin",
        "version": "0.1.0",
        "model_class_name": "None",
        "open_source_license": "Apache-2.0",
        "docs_base_url": "https://docs.nautobot.com",
        "docs_app_url": "https://docs.nautobot.com/projects/plugin-nornir/en/latest",
        "_template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
        "_output_dir": "/opt/ntc/drift-manager/outputs",
        "_repo_dir": "/opt/ntc/drift-manager/outputs/.cookiecutters/cookiecutter-nautobot-app/nautobot-app"
    },
    "base_branch": "next-2.0",
    "remote_name": "origin",
    "pull_request_strategy": "PullRequestStrategy.CREATE",
    "post_actions": [
        "PostAction.BLACK"
    ]
}
```

CLI Arguments:

```
{
    "cookie_dir": "",
    "input": false,
    "json_filename": "",
    "output_dir": "./outputs",
    "push": true,
    "template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
    "template_dir": "nautobot-app",
    "template_ref": "develop",
    "pull_request": null,
    "post_action": [
        "black"
    ],
    "disable_post_actions": false
}
```

### CODE REVIEW

**Comment by snaselj** [comment]
Ready for review @itdependsnetworks

File: nautobot_plugin_nornir/__init__.py [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
 
     name = "nautobot_plugin_nornir"
-    verbose_name = "Nautobot Plugin for Nornir"
+    verbose_name = "Nautobot Nornir Plugin"
     version = __version__
     author = "Network to Code, LLC"
-    description = (
-        "Nautobot App that provides a shim layer to simplify using Nornir within other Nautobot Apps and Nautobot Jobs"
-    )
+    description = "Nautobot Nornir Plugin."
```
Comment (itdependsnetworks) [inline nautobot_plugin_nornir/__init__.py:17]: please revert.

File: pyproject.toml [hunk 2]
```diff
@@ -43,34 +41,34 @@ bandit = "*"
...
+flake8 = "*"
 invoke = "*"
 ipython = "*"
 pydocstyle = "*"
 pylint = "*"
 pylint-django = "*"
-pylint-nautobot = { git = "https://github.com/nautobot/pylint-nautobot.git", branch = "develop" }  # TODO: Point to 2.x version
+pylint-nautobot = "*"
 pytest = "*"
+python-dotenv = "^1.0.0"
```
Comment (itdependsnetworks) [inline pyproject.toml:52]: please revert.

File: pyproject.toml [hunk 3]
```diff
@@ -114,6 +112,11 @@ notes = """,
     XXX,
     """
 
+[tool.pylint-nautobot]
+supported_nautobot_versions = [
+    "1.4.0"
```
Comment (itdependsnetworks) [inline pyproject.toml:117]: what should this be?

File: tasks.py [hunk 4]
```diff
@@ -13,10 +13,15 @@
...
-from distutils.util import strtobool
 
-from invoke import Collection
-from invoke import task as invoke_task
+from dotenv import load_dotenv
+from invoke.collection import Collection
+from invoke.tasks import task as invoke_task
+
+
+def _load_dotenv():
```
Comment (itdependsnetworks) [inline tasks.py:22]: please revert, as mentioned on slack, this shouldn't be used as a standard yet.

**Comment by snaselj** [comment]
Will re-bake it using the latest template based on the latest `develop`.

**Pull request closed without merging**